app/controllers/grocer/exports_controller.rb

```diff
@@ -37,7 +38,7 @@ def set_export
     # Only allow a trusted parameter "white list" through.
     def export_params
       params.require(:export).permit(:pid, :job, :status, :last_error, :last_success, :logfile,
-                                     :page)
+                                     :page, :ark)
```

Comment:
We're passing the ARK here so it can get passed to the export job and added to the Bag metadata.  We use the `pid` as the unique key for retrieving the object from Fedora.